lib/bolt/transport/winrm.rb

```diff
@@ -37,10 +37,13 @@ def initialize
         super
         require 'winrm'
         require 'winrm-fs'
+
+        @transport_logger = Logging.logger[::WinRM]
+        @transport_logger.level = :warn
```

Comment:
Do we want to set warn here since we're not trying to silence net::ssh?

Reply:
Any idea why it was set in the first place? I'm not changing the behavior, just where we set it up.

Reply:
not really. I guess we can leave it for now